tony-core/src/test/java/com/linkedin/tony/TestUtils.java

```diff
@@ -40,6 +40,27 @@ public void testUnzipArchive() {
     } catch (IOException e) {
       fail(e.toString());
     }
+  }
 
+  @Test
+  public void testIsArchive() {
+    ClassLoader classLoader = getClass().getClassLoader();
+    File file1 = new File(classLoader.getResource("test.zip").getFile());
+    File file2 = new File(classLoader.getResource("test.tar").getFile());
+    File file3 = new File(classLoader.getResource("test.tar.gz").getFile());
```

Comment:
Can the test generate these .zip, .tar, and .tar.gz files instead of checking them in? (In general, I think we should reduce the number of binary artifacts checked in to our repo.)

Reply:
We don't have the utils to generate archive files.. also, we need different logic/dependencies to zip/tar/tar.gz files, that is overkill for this.

Reply:
Fair enough